xapian_db.gemspec

```diff
@@ -5,7 +5,7 @@ $:.unshift lib unless $:.include?(lib)
 
 Gem::Specification.new do |s|
   s.name         = %q{xapian_db}
-  s.version      = "1.3.14"
+  s.version      = "1.4.0"
```

Comment:
I think 1.3.15 is fine.
There are no breaking changes. It still works with normal YML files without ERB code